SR2023/2022-09-29-confirm-team-details.md

```diff
@@ -0,0 +1,24 @@
+---
+to: SR2023 sign-ups
+subject: Confirming your details for SR2023
+---
+
+Hello!
+
+Thank you for your interest in Student Robotics 2023.
+
+Before we can confirm your place, we have a few questions and extra pieces of information we need.
+
+1. Can you confirm you are over the age of 18, and officially related to the institution you are bringing a team from (eg teacher at a school, leader at a scout group etc).
+2. Could you provide a second set of contact details for someone at your institution. They should know about the competition, and can be used as a backup contact.
+   1. Name
+   2. Email address
```

Comment:
We should probably ensure that:
- the second contact is another staff member, ideally a head of dept?
- we get phone numbers for both contacts. A useful framing here is probably something like "we will only contact you by phone for urgent things, please ensure it is a number that you would have with you at events"